config/webpack/entries/main.js

```diff
@@ -6,7 +6,58 @@
  */
 
 import '@/public/dist/uib-templates';
+import angular from 'angular';
 
 if (process.env.NODE_ENV === 'production') {
   require('@/public/dist/templates');
 }
+
+/*
+ *  Imports all react components from modules/ and register them as angular components
+ *
+ *  So if you defined modules/users/client/components/TrustrootsGreeting.component.js as:
+ *
+ *    export default function TrustrootsGreeting({ name }) {
+ *      return <p>Hello {name}, from Trustroots!</p>;
+ *    };
+ *
+ *  It will be available to use in (any) angular template as:
+ *
+ *    <trustroots-greeting name="'Donald'"></trustroots-greeting>
+ *
+ *  Uses a webpack require context
+ *  See https://webpack.js.org/guides/dependency-management/#require-context
+ */
+importComponents(require.context('../../../modules/', true, /\.component\.js$/));
+
+function importComponents(r) {
+  r.keys().forEach(path => {
+    const Component = r(path).default;
+    const name = extractComponentNameFromPath(path);
+    if (name !== Component.name) {
```

Comment:
What is the motivation for this check @nicksellen? The application running via `start:prod` fails here and the check doesn't seem to be required. Maybe we could disable this in production env?

Reply:
I changed this in 68dece5. To see the original, see the branch at 5efdfa6.